testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -59,6 +59,16 @@ def setUp(self):
             'acceptor_resnm': np.array(['ALA', 'ALA', 'ALA', 'ALA', 'ALA', 'PRO', 'ALA', 'ALA'], dtype='U4'),
             }
 
+    #issue 1229        
+    def test_get_bonded_hydrogen_list(self, **kwargs):
+    	kw = self.kwargs.copy()
+    	kw.update(kwargs)
+    	h = MDAnalysis.analysis.hbonds.HydrogenBondAnalysis(self.universe, **kw)
+    	self.universe.atoms[0].index = -2
```

Comment:
I'm not sure what you're trying to do here, but you can't set the index of an Atom